Update UG, AboutUs, PPP (#246)
AY2223S1-CS2103T-F12-1/tp: docs/UserGuide.md

good to take note of them before continuing.

### Command-specific terms
1. **Command String**: The text you input into the [command input box](#1-command-input-and-output-boxes), which is

> translated to a Command by Cobb

Reviewer: ```suggestion
translated to a Command by Cobb.
```

2. **Command**: An instruction that is executed by Cobb and translated from your Command String 

> 3. **Syntax**: The manner in which is a Command String is to be typed. Can be thought of as the "format" of the command string.

Reviewer: ```suggestion
3. **Syntax**: The manner in which a Command String is to be typed. Can be thought of as the "format" of the command string.
```

Author: I think the original sentence makes more grammatical sense haha

> 4. **Parameter**: An input to a command. Can be thought of as a piece of information a user provides that determines the

Reviewer 2: I think we can remove this because i changed all "parameter" references to "input" because of what Ms Koh said!

Author: Okay removed the rest of the parameter mentions!

8. **Ascending**: Related to order; from low to high or minimum to maximum.
9. **Descending**: Related to order; from high to low or maximum to minimum. 

### Buyer-specific terms

> 1. **Name**: The name of the buyer.

Reviewer: Actually, isn't `Name` used for the `Owner` class as well?

Author: Changed `Owner` to `Owner Name`!

</div>

### 2. Buyer List
You can find the buyer list located at the left section of the window.<br>
![Buyer List](images/BuyerList.png)<br><br>
The __Buyer List__ displays information regarding buyers who are currently stored in Cobb's database. The buyer list
might not be showing **all* the buyers in the database all the time (see [FAQ](#faq)).

The buyer list can be filtered and modified using commands given in the [features](#features) section of the guide.

### 3. Property List

> You can find the property list located at the right section of Cobb's GUI.

Reviewer: ```suggestion
You can find the property list located at the right section of the window.
```
For consistency!

Author: Removed all mentions of GUI in the UG as well
